public/css/user/home.css

```diff
@@ -3,6 +3,266 @@
+  width: 550px;
+  height: 200px;
+  justify-content: center;
+  align-items: center;
+  font-size: 72px;
+}
+
+.main--header-star {
+  position: absolute;
+  left: 80%;
+  top: 10%;
+  width: 27%;
+  height: 52%;
+}
+
+.main--header-starGreen {
+  position: absolute;
+  clip-path: polygon( 50% 0%, 61% 35%, 98% 35%, 68% 57%, 79% 91%, 50% 70%, 21% 91%, 32% 57%, 2% 35%, 39% 35%);
+  width: 500px;
+  height: 500px;
+  background: rgba(0, 131, 38, 0.9);
+  z-index: 1;
+}
+
+.main--header-starGray {
+  position: absolute;
+  clip-path: polygon( 50% 0%, 61% 35%, 98% 35%, 68% 57%, 79% 91%, 50% 70%, 21% 91%, 32% 57%, 2% 35%, 39% 35%);
+  transform: matrix(0.97, -0.23, 0.23, 0.97, 0, 0);
+  width: 500px;
+  height: 500px;
+  background: #c4c4c4;
+}
+
+.main--header-description {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 200px;
+}
+
+.main--header-description-p {
+  width: 800px;
+  height: 120px;
+  font-family: lato;
+  font-size: 26px;
+  margin-left: 80px;
+  font-weight: 300;
+}
+
+.main--header-description-request {
+  align-self: flex-end;
+  margin-right: 100px;
+  margin-top: 150px;
+}
+
+.main--header-forYou, .main--header-helpOthers {
+  background: #008326;
+  font-size: 30px;
+  width: 200px;
+  height: 60px;
+  border: none;
+  border-radius: 5px;
+}
+
+.main--header-forYouA, .main--header-helpOthersA {
+  color: white;
+  text-decoration: none;
+}
+
+.main--header-ourVision {
+  background-image: linear-gradient( to left bottom, rgba(1, 1, 1, 0.5), rgba(1, 1, 1, 0.5)), url("https://files.gitter.im/ahmedisam99/pYeg/donation.jpg");
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 140px 50px 120px 50px;
+}
+
+.main--header-ourVision-h1, .main--header-ourVision-h1 {
+  font-size: 80px;
+  color: white;
+  font-family: Roboto;
+  font-weight: 300;
+  margin-bottom: 50px;
+}
+
+.main--header-ourVision-p, .main--header-ourVision-p {
+  font-size: 30px;
+  color: #aeffc9;
+  font-family: Roboto;
+  font-weight: 300;
+}
+
+.main--header-how {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 140px 50px 120px 50px;
+}
+
+.main--header-how-h1, .main--header-how-h1 {
+  font-size: 80px;
+  color: #008326;
+  font-family: Roboto;
+  font-weight: 300;
+}
+
+.main--header-how-p, .main--header-how-p {
+  font-size: 30px;
+  color: black;
+  font-family: Roboto;
+  font-weight: 300;
+}
+
+.footer {
+  height: 600px;
+  display: flex;
+  justify-content: space-around;
+  color: white;
+  background: #040404;
+  padding: 50px;
+}
+
+.footer--connectionIcon {
+  width: 80px;
+}
+
+.footer--contactUs-form {
+  display: flex;
+  flex-direction: column;
+  width: 500px;
+}
+
+.footer--contactUs-h1 {
+  color: white;
+  text-align: center;
+  margin-bottom: 50px;
+  font-size: 70px;
+}
+
+.footer--contactUs-formFullName, .footer--contactUs-formEmail, .footer--contactUs-formMessage, .footer--form-registerEmail {
```

Comment:
@aminalakhsham You could have a single class for all of your footer forms, instead of having many classes which all share styles:
``
.footer--form-field {
  ...
}
```
The advantage is, you can add a new form field in your hbs and it will automatically be styled with these styles, without you also having to come into this file and add the new specific classname to this line.
Of course, if you need distinct classnames so that you can select them in the javascript when submitting the form, you could use two classes: one shared class for the form styling, one unique class for getting that field's value:
`<input class="footer--form-field contactUs" ...`

Reply:
nice note i'll take it in mind 
thank you for helpful comment